check_source.py

```diff
@@ -149,9 +150,9 @@ def check_source_submission(self, source_project, source_package, source_revisio
         civs += 'LC_ALL=C perl %s _old %s 2>&1' % (source_checker, target_package)
         p = subprocess.Popen(civs, shell=True, stdout=subprocess.PIPE, close_fds=True)
         ret = os.waitpid(p.pid, 0)[1]
-        checked = p.stdout.readlines()
+        checked = decode_list(p.stdout.readlines())
 
-        output = '  '.join(checked).translate(None, '\033')
+        output = '  '.join(checked).replace('\033', '')
```

Comment:
personally I'd using static method `.translate(str.maketrans('','','\033'))` though, but changed to `replaced()` from `translate()` also ok. :)

Reply:
I wondered what translate actually does - and I would even be more confused with the maketrans version. Because it's not translated, it's plainly removed. So I took the replace version :)

Reply:
yeah, `maketrans` is `(in, out, mapped to None)` :)